lib/GoogleAuthenticator.php

```diff
@@ -103,7 +103,7 @@ public function getUrl($user, $hostname, $secret)
         $args = func_get_args();
         $encoder = 'https://chart.googleapis.com/chart?chs=200x200&chld=M|0&cht=qr&chl=';
         $urlString = '%sotpauth://totp/%s@%s%%3Fsecret%%3D%s%%26issuer%%3D%s';
-        $encoderURL = sprintf($urlString, $encoder, $user, $hostname, $secret, $args[3]);
+        $encoderURL = sprintf($urlString, $encoder, $user, $hostname, $secret, (array_key_exists(3, $args) ? $args[3] : ''));
```

Comment:
@greg0ire OK.
@OskarStark I will make the change once it's clear what you mean exactly... Please elaborate.